### ISSUE TITLE
Image rotation

### ISSUE THREAD

**Issue by ArtStepanyuk** [rap2hpoutre/vue-picture-swipe#7]
Hello Is image rotation supported now? I can't find any information about it in documents

**Comment by rap2hpoutre** [comment]
Hi! Thank you for your contribution! Do you mean: does the image rotate if the phone is rotated?

**Comment by ArtStepanyuk** [comment]
No, I mean actual image rotations line this... (I edited your code a bit to check it locally)

![1](https://user-images.githubusercontent.com/9801069/40301963-5d7534e8-5cf6-11e8-8c4e-5cc4b6d6022b.png)
![2](https://user-images.githubusercontent.com/9801069/40301966-5d9724a4-5cf6-11e8-8300-785ab0d9e9f9.png)

**Comment by rap2hpoutre** [comment]
This package is a wrapper for PhotoSwipe. It seems it's not possible with PhotoSwipe options (see https://github.com/dimsemenov/PhotoSwipe/issues/1141 and http://photoswipe.com/documentation/options.html), but maybe there is something to tackle to make it work. So maybe you should try to make it work with PhotoSwipe, then update the HTML of this package if required (and submit a PR), then use `options`:

```vue
<vue-picture-swipe :options="{myRotationOption: false}"></vue-picture-swipe>
```

**Comment by ArtStepanyuk** [comment]
I have added functionality to the vue part (I'm not very familiar with jquery plugin itself) I also pulled material design for icons if you want I can create PR so you can review functionality since I find it quite useful and review the code itself. 

![3](https://user-images.githubusercontent.com/9801069/40303516-b6378018-5cfb-11e8-9720-2fe6d23008ad.png)


https://github.com/rap2hpoutre/vue-picture-swipe/pull/8

Please check if this is an acceptable solution

Thank you

**Comment by rap2hpoutre** [comment]
Fixed via #8 

Thanks again!

**Comment by rap2hpoutre** [comment]
Available in v0.3.0

**Comment by ArtStepanyuk** [comment]
I was glad to help thank you for amazing lib :)
